app/views/foods/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1 class="text-center py-3">Add New Food</h1>
+
+<%= form_with(model: @food, url: foods_path, method: :post) do |form| %>
+  <% if @food.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@food.errors.count, "error") %> prohibited this food item from being saved:</h2>
+
+      <ul>
+        <% @food.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="form-group my-3">
+    <%= form.label :name %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :measurement_unit %>
+    <%= form.text_field :measurement_unit, class: "form-control" %>
+  </div>
+
+  <div class="form-group my-3">
+    <%= form.label :price %>
+    <%= form.text_field :price, class: "form-control" %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit "Submit", class: "btn btn-primary my-3" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Great work so far but this requirement is for the below 3 members group :thinking: Now the issue is that the inventory food form is dependent on this form which is totally against the requirements :fearful: Kindly remove this form and make the inventory food form independent :+1: to follow the project requirements :blush: .